package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "start": "bun run build && bun --hot run dev-server.ts",
     "dev": "bun run start",
-    "build": "bunx make-vfs ./symbols --content-format import-default --outfile ./generated/symbols-index.ts && tsup-node ./index.ts --format esm --dts --sourcemap && bun run scripts/generate-base-symbol-names.ts",
+    "build": "bunx make-vfs ./symbols --content-format import-default --outfile ./generated/symbols-index.ts && tsup-node ./index.ts --format esm --dts --sourcemap && bun run scripts/generate-base-symbol-names.ts && bun test",
```

Comment:
Test should be a separate step than build, however it is ok to prevuild before testing

Reply:
I wanted to make sure everyone will run tests before pushing their PR, should I do that in another way?

Reply:
Yea i think if the workflow is in place people will be reminded to run their tests (because it'll fail). But I think we should be pretty explicit with scripts, if a build script runs tests then we might run tests when e.g. building the package for release. Which is fine but it might confuse somebody. Principle of least surprise

Reply:
Added a step on readme to remind people to run them it 0eaf7b25cab906afe54bcdb6afba2c40bebce633